plat/layerscape/common/ls_topology.c

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2018, ARM Limited and Contributors. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <arch.h>
+#include <plat_arm.h>
+#include <platform_def.h>
+
+/*******************************************************************************
+ * This function validates an MPIDR by checking whether it falls within the
+ * acceptable bounds. An error code (-1) is returned if an incorrect mpidr
+ * is passed.
+ ******************************************************************************/
+int ls_check_mpidr(u_register_t mpidr)
+{
+	unsigned int cluster_id, cpu_id;
+	uint64_t valid_mask;
+
+#if ARM_PLAT_MT
```

Comment:
`ARM_PLAT_MT` only applies to Arm standard platforms. In your case, you should remove the part of the code that doesn't apply to you. If your CPUs are multithreaded, leave the part with the `pe_id`. If they are not, leave the part without `pe_id`. As far as I can see, you use Cortex-A53. This means that you don't have `pe_id`, and you should remove lines 21-30 and 50-53.

Reply:
Done, thanks.